clang/test/SemaSYCL/array-kernel-param-neg.cpp

```diff
@@ -1,43 +1,49 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-device -fcxx-exceptions -Wno-sycl-2017-compat -verify -fsyntax-only %s
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -fcxx-exceptions -sycl-std=2020 -verify -fsyntax-only %s
 
 // This test checks if compiler reports compilation error on an attempt to pass
 // an array of non-trivially copyable structs as SYCL kernel parameter or
 // a non-constant size array.
 
-struct B {
+#include "Inputs/sycl.hpp"
+
+cl::sycl::queue q;
```

Comment:
I guess you can just use `sycl::` instead of `cl::sycl::` everywhere since you use `sycl-std=2020`

Reply:
I will have to update the mock "sycl.hpp" header to remove "cl" namespace from it.
Wouldn't it cause issues for tests using the older sycl spec?

Reply:
I don't expect issues if `cl` namespace is inlined (the same way we do in DPC++ library implementation).

Reply:
@bader Thanks for the tip. I have inlined "`cl`" namespace in the mock header and updates tests to use just "`sycl::`".